### PR TITLE
[author] added jplist v5.2.0 and auto update config

### PR DESCRIPTION
**Pull request by no81no**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/no81no/jplist
 * Official website (optional, not the repository): http://jplist.com
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 303
   - Count of watchers: 34
   - Count of forks: 120

# Essential checklist
 * [v] I'm the author of this library
   * [v] I would like to add link to the page of this library on CDNJS on website / readme
 * [v] This lib was not found on cdnjs repo
 * [v] No already exist / duplicated issue and PR
 * [v] The lib has notable popularity
   * [v] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]

# Auto-update checklist
 * [v] Has valid tags for each versions (for git auto-update)
 * [v] Auto-update setup
 * [v] Auto-update target/source is valid.
 * [v] Auto-update filemap is correct.

# Git commit checklist
 * [v] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [v] The parent of the commit(s) in the PR is not old than 3 days.
 * [v] Pull request is sending from a non-master branch with meaningful name.
 * [v] Separate unrelated changes into different commits.
 * [v] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [v] Close corresponding issue in commit message
 * [v] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jplist/package.json [hunk 1]
```diff
@@ -17,8 +17,48 @@
...
     "url": "https://github.com/no81no/jplist"
   },
-  "license": "MIT",
-  "authors": [
-    "Miriam Zusin <no81no@gmail.com>"
-  ]
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/no81no/jplist.git",
+    "basePath": "",
```
Comment (PeterDaveHello) [inline ajax/libs/jplist/package.json]: We should set basePath to `"dist"` here and remove the `dist/` prefix in filemap. If you want to include whole `dist` folder, please use single rule `"**/*"`, thanks.

Comment (no81no) [inline ajax/libs/jplist/package.json]: It's fixed, 
thank you.

**Comment by PeterDaveHello** [comment]
So what about this:
`"**/*.+(css|js|map)"`

Easier and flexible.

File: ajax/libs/jplist/package.json (hunk 1 above)
Comment (no81no) [inline ajax/libs/jplist/package.json]: Awesome, really convenient. I've added it instead of files list.

File: ajax/libs/jplist/package.json [hunk 2]
```diff
@@ -21,44 +21,6 @@
...
-      "css/jplist.pagination-bundle.min.css",
-      "css/jplist.preloader-control.min.css",
-      "css/jplist.radio-buttons-dropdown.min.css",
-      "css/jplist.sort-bundle.min.css",
-      "css/jplist.sort-buttons-control.min.css",
-      "css/jplist.start-rating-control.min.css",
-      "css/jplist.textbox-filter.min.css",
-      "css/jplist.views-control.min.css"
-    ]
+    "files": "**/*.+(css|js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/jplist/package.json]: Still need to be in an array

File: ajax/libs/jplist/package.json [hunk 3]
```diff
@@ -17,8 +17,10 @@
...
   },
-  "license": "MIT",
-  "authors": [
-    "Miriam Zusin <no81no@gmail.com>"
-  ]
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/no81no/jplist.git",
+    "basePath": "dist",
+    "files": "**/*.+(css|js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/jplist/package.json]: Please put `"**/*.+(css|js|map)"` in an array.

File: ajax/libs/jplist/package.json [hunk 4]
```diff
@@ -17,8 +17,10 @@
     "type": "git",
     "url": "https://github.com/no81no/jplist"
   },
-  "license": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/jplist/package.json]: What's the reason why to remove license and author info?

File: ajax/libs/jplist/package.json (hunk 2 above)
Comment (no81no) [inline ajax/libs/jplist/package.json]: Should it be like this? 
"files": ["**/*.+(css|js|map)"]

Comment (PeterDaveHello) [inline ajax/libs/jplist/package.json]:
```js
"files": [
  "**/*.+(css|js|map)"
]
```

**Comment by PeterDaveHello** [comment]
@no81no sorry that i didn't notice that you put the files in wrong path, in the commit, please just put the css files in `css` and so does js files as you did in your repo, please don't forget to update filename field to reflect this change, thanks.

**Comment by no81no** [comment]
Done, I hope it's correct :)

**Comment by no81no** [comment]
Thank you!

**Comment by PeterDaveHello** [comment]
No problems.

**Pull request merged**